root/components/EntityHeader.js

```diff
@@ -12,16 +12,17 @@ import * as React from 'react';
 import EntityLink from '../static/scripts/common/components/EntityLink';
 
 import EntityTabs from './EntityTabs';
+import typeof EntityTabLink from './EntityTabLink';
 import SubHeader from './SubHeader';
 
 type Props = {
-  +editTab?: React.Node,
+  +editTab?: React.Element<EntityTabLink>,
```

Comment:
Here's an example of where it might be useful to specify the type of element we want.